examples/scaling_example/scaling.c

```diff
@@ -374,14 +372,9 @@ thread_main_loop(struct onvm_nf_local_ctx *nf_local_ctx) {
                         pktsTX[tx_batch_size++] = pkts[i];
                 }
 
-                if (unlikely(tx_batch_size > 0 && rte_ring_enqueue_bulk(tx_ring, pktsTX, tx_batch_size, NULL) == 0)) {
-                        nf->stats.tx_drop += tx_batch_size;
-                        for (j = 0; j < tx_batch_size; j++) {
-                                rte_pktmbuf_free(pktsTX[j]);
-                        }
-                } else {
-                        nf->stats.tx += tx_batch_size;
-                }
+                /* Process all packet actions */
+                onvm_pkt_process_tx_batch(nf->nf_tx_mgr, pktsTX, tx_batch_size, nf);
+                onvm_pkt_flush_all_nfs(nf->nf_tx_mgr, nf);
```

Comment:
Would a if condition before `onvm_pkt_flush_all_nfs` be good?
Since `onvm_pkt_process_tx_batch` internally calls onvm_pkt_flush_all_nfs` when the tx_batch_size is 32, I think it would be better to add a condition to check if tx_batch_size is not 32 before flushing again.

Reply:
yes, good catch